src/templates/static/legal/tac_sections/biel.jsx

```diff
@@ -2,50 +2,92 @@ import React from 'react';
+        <p>{it.L('These ST&Cs begin to apply on the date BIEL opens the client\'s account, and, for any new versions after that, on the date BIEL notifies the client.')}</p>
 
-        <h2 data-anchor='about-us-and-our-regulator'>{it.L('About Us and Our Regulator')}</h2>
+        <h2 data-anchor='about-us-and-our-regulator'>{it.L('About BIEL and BIEL\'s regulator')}</h2>
         <p>{it.L('BIEL is a company established in Malta under registration number C70156 and having its registered office at Mompalao Building, Suite 2, Tower Road, Msida MSD1825, Malta.')}</p>
-        <p>{it.L('We are authorised and regulated by the Malta Financial Services Authority ("MFSA") under the Investment Services Act (Cap. 370, Laws of Malta) (the "Act") to carry on investment business.  We are authorised by the MFSA to deal on own account (i.e. offer and act as counterparty to trades) and to execute orders on behalf of other clients, both services are in relation to financial derivative products relating to foreign exchange, indices and other financial products or assets ("Financial Products"). We are not authorised to and do not offer or provide investment advice or any other investment services other than those described above. Should you feel that you require investment advice or investment services other than those described above then you should contact an independent financial advisor or firm.')}</p>
-        <p>{it.L('In providing these services, we are bound by the Act, any applicable regulations, bye-laws, licence conditions, guidelines, exchange requirements and other provisions or market practices (the "Rules"). In the event of conflict between these ST&Cs and the Rules, the latter should prevail.')}</p>
+        <p>{it.L('BIEL is authorised and regulated by the Malta Financial Services Authority ("MFSA") under the Investment Services Act (Cap. 370, Laws of Malta) (the "Act") to carry on investment business. BIEL is authorised by the MFSA to deal on own account (i.e. offer and act as counterparty to trades) and to execute orders on behalf of other clients; both services are in relation to financial derivative products relating to foreign exchange, indices and other financial products or assets ("Financial Products"). BIEL is not authorised to and does not offer investment, financial, legal, tax, regulatory, and/or other types of advice or state an opinion in relation to a Transaction under any circumstances. Clients should get independent advice from an investment adviser if they have any doubts about dealing in financial products.')}</p>
+        <p>{it.L('From time to time, BIEL may decide to provide the Client with written or video information, which BIEL may publish on its website or provide to the Client in any other manner. BIEL will endeavour to ensure the accuracy and completeness of this information, but it will not constitute independent investment research or investment advice provided by BIEL to the Client.')}</p>
```

Comment:
```suggestion
        <p>{it.L('From time to time, BIEL may decide to provide the Client with written or video information, which BIEL may publish on its website or provide to the Client in any other manner. BIEL will endeavour to ensure the accuracy and completeness of this information, but it will not constitute independent investment research or investment advice provided by BIEL to the client.')}</p>
```